Lesson 5

LizavetBa opened the pull request:
finished fifth HT

LizavetBa committed:
added new part of project
added new container "Blue"

LizavetBa committed:
added fonts.css

almuly commented on index.html:
```diff
@@ -1,17 +1,59 @@
+        <p> Real guest reviews</p>
+    </div>
+</div>
+<div class="Blue">
```
in this case for items in columns better to use ul and li tags

LizavetBa replied:
added lists

almuly commented on index.html:
```diff
@@ -1,17 +1,59 @@
+        <p> Real guest reviews</p>
+    </div>
+</div>
+<div class="Blue">
```
bad neme for class, it doesn't described for what you use it. better .background_wrapper
and never use capital letter in nemes of classes

LizavetBa replied:
renamed

almuly commented on styles/style.css:
```diff
@@ -0,0 +1,123 @@
+    color: white;
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 1440px;
```
for align conten in the center of block use margin: 0 auto

LizavetBa replied:
added margin

almuly commented on styles/style.css:
```diff
@@ -0,0 +1,123 @@
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 1440px;
+    padding: 120px 0 200px 104px;
```
after aligning you need to check this paddings

LizavetBa replied:
correct paddings

almuly commented on styles/style.css:
```diff
@@ -0,0 +1,123 @@
+    flex-wrap: wrap;
+    max-width: 1440px;
+    padding: 120px 0 200px 104px;
+    background-color: #10367c;
```
according to design your background must  cover all scrren, so for that you can create new div with class and transfer backgrounfd color there

almuly commented on styles/style.css:
```diff
@@ -0,0 +1,123 @@
+}
+
+h1 {
+    display: flex;
```
Wrong font-weight

LizavetBa replied:
added font-weight 500

almuly commented on styles/style.css:
```diff
@@ -0,0 +1,123 @@
+    display: flex;
+    justify-content: center;
+    width: 306px;
+    height: 47px;
```
don't use fix height, better to add padding between title and line

LizavetBa replied:
removed height and added padding

almuly commented on styles/style.css:
```diff
@@ -0,0 +1,123 @@
+    justify-content: center;
+    width: 306px;
+    height: 47px;
+    margin: 120px 567px 0 567px;
```
Unneccessary, you can yse paading top and thats all

LizavetBa replied:
done

almuly commented on styles/style.css:
```diff
@@ -0,0 +1,123 @@
+    height: 176px;
+}
+
+.headphones img {
```
add img wrapper for all images wuth property width:100px, height, 100px, then for images you can use property height auto and you didn't  need to describe five different classes

LizavetBa replied:
but every img has its own size and in Figma there diffirent sizes for each img

LizavetBa committed:
rename class .Blue to background_wrapper
added margin 0 auto for block headphones/chat/ok/laptop p
checked the paddings after aligning
instead height to h1 used padding-bottom

LizavetBa committed:
added list for classes down

LizavetBa committed:
added font-weight 500 for h1

LizavetBa committed:
added block footer
fixed alignment

LizavetBa commented:
fixed

LizavetBa committed:
fixed problem with pics
